host-interaction/file-system/write/write-file-on-windows.yml

```diff
@@ -24,11 +24,10 @@ rule:
               - number: 0x40000000 = GENERIC_WRITE
               - number: 0x2 = FILE_WRITE_DATA
               - match: create or open file
-          - call:
-            - or:
-              - number: 0x40000000 = GENERIC_WRITE
-              - number: 0x2 = FILE_WRITE_DATA
-              - match: create or open file
+          - or:
+            - number: 0x40000000 = GENERIC_WRITE
+            - number: 0x2 = FILE_WRITE_DATA
+            - match: create or open file
```

Comment:
here's where things can get tricky: the scope of `create or open file` is (fortunately) also `call`, but in some other rules is a larger scope (like `thread`), which is impossible to match.

so, i think this change is fine.

but we should be aware that in other scenarios, there's some tricky things we might have to fix.